### PR TITLE
expanded episode id in url

### PR DESCRIPTION
**Pull request by dominik-git**
You should to see episodeID in a url address when you navigate to episode in Home page and you press enter.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -11,5 +11,6 @@ ReactDOM.render((
     <Route path="/series/:id" component={SeriesDetail} />
     <Route path="/" component={App} />
     <Route path="/:selectedEpisodeId" component={App} />
```
Comment (garySiladi) [inline src/index.js]: I think we don't need this line here ... just replace the functionality with your URL param

**Comment by garySiladi** [comment]
Are the tests fully covered?

**Comment by dominik-git** [comment]
yes

2017-06-09 8:57 GMT+02:00 garySiladi <notifications@github.com>:

File: src/index.js [hunk 2]
```diff
@@ -10,6 +10,6 @@ ReactDOM.render((
   <Router history={browserHistory}>
     <Route path="/series/:id" component={SeriesDetail} />
     <Route path="/" component={App} />
-    <Route path="/:selectedEpisodeId" component={App} />
+    <Route path="/expandedEpisode/:id" component={App} />
```
Comment (varholak-peter) [inline src/index.js]: path="/:expandedEpisode

**Pull request merged**